base_layer/wallet_ffi/src/lib.rs

```diff
@@ -4836,6 +4842,31 @@ pub unsafe extern "C" fn wallet_import_utxo(
         return 0;
     }
 
+    if metadata_signature.is_null() {
+        error = LibWalletError::from(InterfaceError::NullError("metadata_signature".to_string())).code;
+        ptr::swap(error_out, &mut error as *mut c_int);
+        return 0;
+    }
+
+    if sender_offset_public_key.is_null() {
+        error = LibWalletError::from(InterfaceError::NullError("sender_offset_public_key".to_string())).code;
+        ptr::swap(error_out, &mut error as *mut c_int);
+        return 0;
+    }
+
+    if script_private_key.is_null() {
+        error = LibWalletError::from(InterfaceError::NullError("script_private_key".to_string())).code;
+        ptr::swap(error_out, &mut error as *mut c_int);
+        return 0;
+    }
+
+    if features.is_null() {
+        *features = OutputFeatures::with_maturity(0);
+    }
```

Comment:
This should be an error condition please, in preparation for the upcoming recovery byte. Default features will never be acceptable again. It will either be a function of the commitment alone, or of the commitment and a recovery byte key.